doc/nrf/releases_and_maturity.rst

```diff
@@ -27,3 +27,15 @@ If an issue is found in a release after it has taken place, those issues are lis
    releases_and_maturity/repository_revisions
    releases_and_maturity/software_maturity
    releases_and_maturity/known_issues
+
+Migration guides
+****************
+
+The |NCS| provides migration guides for all minor releases, in order to assist user's transition from the previous release.
```

Comment:
We intend to provide the guides for major releases too right?

Reply:
Modified